configure.ac

```diff
@@ -6231,6 +6264,14 @@ fi
 
 if test "x$with_libdpdk" = "xyes"
 then
+  plugin_dpdkevents="no (DPDK version < 16.07)"
```

Comment:
Ah yes. The thing is, these runtime tests are slow, so perhaps you could abuse the preprocessor and do a compile time test like:
```
#if RTE_VERSION < RTE_VERSION_NUM(16,7,0,0)
#error "version too old"
#endif
```